COMETwebapp/Components/Shared/ParameterTypeEditors/TimeOfDayParameterTypeEditor.razor

```diff
@@ -0,0 +1,35 @@
+﻿<!------------------------------------------------------------------------------
+Copyright (c) 2023 RHEA System S.A.
+
+Authors: Justine Veirier d'aiguebonne, Sam Gerené, Alex Vorobiev, Alexander van Delft, Jaime Bernar
+
+This file is part of COMET WEB Community Edition
+The COMET WEB Community Edition is the RHEA Web Application implementation of ECSS-E-TM-10-25 Annex A and Annex C.
+
+The COMET WEB Community Edition is free software; you can redistribute it and/or
+modify it under the terms of the GNU Affero General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+
+The COMET WEB Community Edition is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see http://www.gnu.org/licenses/.
+------------------------------------------------------------------------------->
+
+@{
+    var values = this.ViewModel.ValueSet.ActualValue.First().Split(':');
+        
+    int.TryParse(values[0], out var hours);

```

Comment:
What if these tryparses fail? Some error handling?

Reply:
This is something contemplated in #199 but for the moment there is no validation implemented